### PR TITLE
Very basic task to export firms and advisers to csv

### PR DESCRIPTION
**Pull request by lshepstone**
Could probably be improved with a dedicated `Exporter` that can be used by other tasks later.

### CODE REVIEW

**Comment by asmega** [comment]
:+1:

**Comment by jongilbraith** [comment]
:+1:

File: lib/tasks/export.rake [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        'Pension Transfer Percent',
+        'Long-Term Care Percent',
+        'Equity Release Percent',
+        'Inheritance Tax and Estate Planning Percent',
+        'Wills and Probate percent',
+        'Other Percent',
+        'Investment Sizes'
+      ]
+
+      Firm.order(id: :asc).each do |firm|
```
Comment (benlovell) [inline lib/tasks/export.rake]: Maybe include `where.not(email_address: nil)` to only dump those firms that completed the questionnaire? Remembering that partially entered firms are also in the database. If they want the partially entered firms in this dataset then obviously leave as-is.

**Comment by benlovell** [comment]
Nice job. Just a small question/comment.

File: lib/tasks/export.rake (hunk 1 above)
Comment (lshepstone) [inline lib/tasks/export.rake]: Seems Kemi and Mark were happy with the export yesterday, and Kemi would like us to keep the partial records in.

**Pull request merged**